### PR TITLE
Adds Basic Handholding

### PR DESCRIPTION
**Pull request by Geevies**
By request. More features might follow, idk. Currently it plays off the dancing thing that makes people face eachother.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/mob/mob_grab.dm [hunk 1]
```diff
@@ -72,6 +72,8 @@
 				G.dancing = 1
 				G.adjust_position()
 				dancing = 1
+				if((G.assailant.zone_sel.selecting == BP_L_HAND || G.assailant.zone_sel.selecting == BP_R_HAND) && (assailant.zone_sel.selecting == BP_L_HAND || assailant.zone_sel.selecting == BP_R_HAND))
+					assailant.visible_message(span("good", "[assailant.name] and [G.assailant.name] holds hands."), range = 3)
```
Comment (OneOneThreeEight) [inline code/modules/mob/mob_grab.dm]: hold hands*. It's grammatically correct, ok.

Comment (Geevies) [inline code/modules/mob/mob_grab.dm]:
```suggestion
					assailant.visible_message(span("good", "[assailant.name] and [G.assailant.name] hold hands."), range = 3)
```

**Comment by BurgerLUA** [comment]
a pr that was added ironically on citadel is now added unironically on aurorastation

what a time to be alive

**Comment by NonQueueingMatt** [comment]
After a short discussion, this will not be merged.

**Pull request closed without merging**